modules/tinymce/src/core/test/ts/browser/DisabledModeTest.ts

```
@@ -0,0 +1,531 @@
+
+    const excludeReadOnlyEnabledButton = Arr.filter(toolbarButtons, (btn) => btn.name !== 'print');
+    const excludeNonEditableRootButton = Arr.filter(toolbarButtons, (btn) => btn.name !== 'print' && btn.name !== 'searchreplace');
+
+    const hook = TinyHooks.bddSetup<Editor>({
+      base_url: '/project/tinymce/js/tinymce',
+      disabled: true,
+      plugins: 'searchreplace',
+      toolbar: Arr.map(allButtons, (button) => button.name).join(' '),
+    }, []);
+
+    afterEach(() => hook.editor().options.set('disabled', true));
```

Biome

[error] Disallow duplicate setup and teardown hooks.

Disallow afterEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)